### PR TITLE
[DOCS] Update docs dependency versions and remove unneeded arguments

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
## What this PR does / why we need it

The dependencies for generating the docs were quite out of date. Updating them introduced a change that removed a mkdocs CLI parameter that is now handled in the config file.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
Ah, the theme changes slightly, I am not sure if that is an issue or not? As far as i could tell, it changes the disclosure menu look, and the logo in the top left (which should probably be updated anyway).

File: docs/Dockerfile [hunk 1]
```diff
@@ -10,8 +10,7 @@ EXPOSE 8000
...
 # by default to reduce size
 RUN pip install \
-    mkdocs==0.17.3 \
-    pymdown-extensions==6.0 \
-    mkdocs-material==2.7.3
+    nltk==3.4.5 \
+    mkdocs-material==5.5.3
 RUN apk add --no-cache git-fast-import openssh-client
-ENTRYPOINT [ "/bin/ash", "-c" ]
+ENTRYPOINT [ "/bin/ash", "-c" ]
```
Comment (arnikola) [inline docs/Dockerfile:16]: nit: newline

Comment (ChrisChinchilla) [reply docs/Dockerfile:16]: OK, most of my editors remove it ;)

Comment (arnikola) [reply docs/Dockerfile]: The red cross at the end really takes away from the aesthetics of the entire review 😜

**Pull request merged**